app/frontend/wallet/keypass-json.js

```diff
@@ -102,23 +110,29 @@ async function isWalletExportEncrypted(walletExportObj) {
 }
 
 async function importWalletSecret(walletExportObj, password) {
-  const {passwordHash, walletSecret} = parseWalletExportObj(walletExportObj)
+  const {passwordHash, walletSecretDef} = parseWalletExportObj(walletExportObj)
```

Comment:
also I would call this function `importWalletSecretDef` for the sake of consistency